src/osd/osd_types.cc

```diff
@@ -5878,3 +5878,33 @@ ostream& operator<<(ostream& out, const store_statfs_t &s)
       << ")";
   return out;
 }
+
+void OSDOp::clear_data(vector<OSDOp>& ops)
+{
+  for (unsigned i = 0; i < ops.size(); i++) {
```

Comment:
exactly.

> I just feel that it might be cleaner to implement OSDOp related function in OSDOp, since there should be no caller want to clear just one OSDOp data.

the same to me. that's why i think this should be a method of `OSDOp`.